libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -171,6 +171,12 @@ void AP_CRSF_Telem::process_rf_mode_changes()
     if (_telem_last_report_ms == 0 && !uart->is_dma_enabled()) {
         gcs().send_text(MAV_SEVERITY_WARNING, "CRSF: running on non-DMA serial port");
     }
+    // note if option was set to show LQ in place of RSSI
+    bool current_lq_as_rssi_active = bool(rc().use_crsf_lq_as_rssi());
+    if(_telem_last_report_ms == 0 || _noted_lq_as_rssi_active != current_lq_as_rssi_active){
+        _noted_lq_as_rssi_active = current_lq_as_rssi_active;
+        gcs().send_text(MAV_SEVERITY_INFO, "CRSF: RSSI is showing %s", current_lq_as_rssi_active ? "LQ" : "RSSI");
```

Comment:
I feel "RSSI is showing RSSI" is a bit confusing, maybe "signal strength is shown as RSSI" or something